Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -29,7 +29,10 @@ There are usually two release engineers working at any given time. It goes witho
 
 ## 2. Release step-by-step
 
-### Phase 1: Initiation
+### Phase 1: Pre-release considerations
+1. To help expedience of the release, any required feature flag changes should be done prior to the 2am cut-off on the Friday. If this should be missed, a concession can be made so that the changes will be allowed to be made during the Friday. However, this should not be a regular occurance as it requires intervention from the assigned release engineers to create new builds.
```

Comment:
```suggestion
1. To help expedite the release, any required feature flag changes should be done prior to the 2am cut-off on the Friday. If these are missed, a concession can be made so that the changes will be allowed to be made during the Friday. However, this should not be a regular occurrence as it requires intervention from the assigned release engineers to create new builds.
```